game/factions/faction.py

```diff
@@ -300,6 +300,12 @@ def apply_mod_settings(self, mod_settings: ModSettings) -> None:
             self.remove_aircraft("Su-57")
         if not mod_settings.ov10a_bronco:
             self.remove_aircraft("Bronco-OV-10A")
+        if not mod_settings.fa18e_superhornet:
```

Comment:
Aren't these all the same mod? If so, just one checkbox.

Reply:
checkboxes changed